monkey/infection_monkey/Pipfile

```diff
@@ -4,6 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+altgraph = "*"
```

Comment:
You'll also need to add the `Pipfile.lock` file to ensure the same version of altgraph is always used.